### DISCUSSION TITLE
VCARD

### DISCUSSION THREAD

**Discussion by Krasheq** [lyqht/mini-qr#60] [Ideas]
Hello! It would be fantastic to have a VRACD option added, not just a link. VCARD is a highly informative and useful method for saving contact information via a QR code.

**Comment by Krasheq** [comment]
example: https://www.qr-code-generator.com/solutions/vcard-qr-code/

**Reply by lyqht** [reply]
done!

**Comment by Martin511** [comment]
I agree.
A VCARD would be great.
